packages/common/src/interfaces/paginationCursorChangedArgs.interface.ts

```diff
@@ -0,0 +1,15 @@
+import { PaginationChangedArgs } from './paginationChangedArgs.interface';
```

Comment:
can you add `type` to this import, there are some benefits in adding them, you can see this SO for more info https://stackoverflow.com/a/64243357/1212166